src/patternfly/components/DataList/data-list.scss

```diff
@@ -114,30 +121,15 @@
   @include pf-t-light;
 
   list-style-type: disc;
-  background-color: var(--pf-c-data-list--BackgroundColor);
```

Comment:
would this be breaking?

Reply:
I don't think so since if someone had defined this as a custom color, the component should still present itself the same visually. Same with moving the pseudo element from the list `__item` element to 2 pseudo elements of the `__item-row` and `__expandable-content` elements.